frontend/dispatcher.lua

```diff
@@ -80,7 +80,9 @@ local settingsList = {
     touch_input_off = {category="none", event="IgnoreTouchInput", arg=true, title=_("Disable touch input"), device=true},
     toggle_touch_input = {category="none", event="IgnoreTouchInput", title=_("Toggle touch input"), device=true, separator=true},
     ----
-    swap_page_turn_buttons = {category="none", event="SwapPageTurnButtons", arg=true, title=_("Invert page turn buttons"), device=true, condition=Device:hasKeys(), separator=true},
+    swap_left_page_turn_buttons = {category="none", event="SwapPageTurnButtons", args={true, "left"}, title=_("Invert left-side page-turn buttons"), device=true, condition= Device:hasDPad() and Device:useDPadAsActionKeys()},
```

Comment:
I doubt if using `args` is correct.
https://github.com/koreader/koreader/blob/491e24ea25e162b2f25e854232603eec986c7f56/frontend/dispatcher.lua#L1167-L1169
Does it work?

Reply:
Huh, good point. I only checked that there were other actions specifying `args`, and assumed it was meant for the Event, but it doesn't look like that's quite what it's for (ConfigDialog/Option stuff?)

Reply:
yes, I also only checked that it inverts the right side one, what I didn't notice was the it was in fact inverting both sets.

Reply:
in this case, @NiLuJe given as how you didn't want to hard code the `show_notification` the other day, we could remove it and send only the side argument. you can fix the sources later

btw, thanks @hius07 for that catch.